src/subcommands/subcommand.go

```diff
@@ -9,13 +10,15 @@ import (
 	"path"
 	"strings"
 
+	db "accretional.com/semantifly/database"
 	pb "accretional.com/semantifly/proto/accretional.com/semantifly/proto"
+	"github.com/jackc/pgx/v5"
 	"google.golang.org/grpc"
 )
 
 type SubcommandInfo struct {
 	Description string
-	Execute     func([]string)
+	Execute     func(context.Context, db.PgxIface, []string)
```

Comment:
pretty sure you can just leave this the way it was and make context/conn package-scoped

Reply:
Hmm... I'll check it out